.github/workflows/image.yaml

```diff
@@ -53,12 +53,12 @@ jobs:
           - ispr: true
             dist: ubuntu20.04
           - ispr: true
-            driver: 470.223.02
+            driver: 470.256.02
```

Comment:
Question -- do we want to build all active datacenter branches on PRs? Currently the actions that ran only built for R550, so we can't verify that the version bump for R470 and R535 builds okay.

Reply:
Yeah, I think that's a good idea

Reply:
I can add those changes to this PR

Reply:
This is done, please review now